tests/features/hashing/test_hashers.py

```diff
@@ -31,3 +31,12 @@ def test_argon2_needs_rehash(self):
         # Here argon2 method is invoked without custom options and will
         # use default argon2 configuration, so rehash will be needed
         assert Hash.needs_rehash(hashed, driver="argon2")
+
+    def test_should_return_hash_content_as_string(self):
+
+        drivers = ["argon2", "bcrypt"]
+
+        for driver in drivers:
+            with self.subTest(driver=driver):
```

Comment:
Here what don't you just do ?
```python
        for driver in drivers:
                hashed = Hash.make("masonite", driver=driver)
                assert type(hashed) == str
```
🤔 Did I miss something ?

Reply:
With subTest ( and using driver as parameter ) let us know which driver fail during run.

Reply:
oh okay !